scheduling_strawman.py

```diff
@@ -0,0 +1,99 @@
+"""
+The scheduler is a very useful tool. It requires the input of observing blocks
+and the time_range. Observing blocks consist of a target, a duration and constraints.
+"global" constraints can also be defined and can significantly speed up the scheduler.
+"""
+
+# a few packages are needed to set up the proper objects
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.time import Time
+import astroplan
+from astroplan.constraints import *
+from astroplan import Observer, FixedTarget
+from astroplan.scheduling import *
+
+# create the time frame during which you observing
+schedule_start = Time.now()+4*u.hour
+schedule_end = schedule_start+24*u.hour
```

Comment:
this is also super-minor... but the PEP8 style guidelines generally suggest spaces around operators.  It's totally a matter of opinion, but I find this often makes things clearer around ``+``, because that's how we tend to write equations on paper.  So I would do ``schedule_start + 24*u.hour`` here instead of ``schedule_start+24*u.hour``